.rubocop_todo.yml

```diff
@@ -51,6 +51,7 @@ Metrics/ClassLength:
     - 'app/controllers/planning_applications_controller.rb'
     - 'app/models/document.rb'
     - 'app/models/planning_application.rb'
+    - 'app/services/planning_application_creation_service.rb'
```

Comment:
How about just adding `# rubocop:disable Metrics/ClassLength` to the `class` line in that file? I dream of someday being able to delete `.rubocop_todo.yml`. 😭